src/components/NewMessageForm/AdvancedInput/AdvancedInput.vue

```diff
@@ -216,7 +216,7 @@ export default {
 	watch: {
 		text(text) {
 			this.$nextTick(() => {
-				this.$emit('update:contentEditable', this.$refs.contentEditable.cloneNode(true))
+				this.$('update:contentEditable', this.$refs.contentEditable.cloneNode(true))
```

Comment:
is there a difference between `$emit` and `$`?

Reply:
yes, afaik `$` is for the instance properties. You need to specify which one after it